aws.sh

```diff
@@ -8,10 +8,18 @@ set -o pipefail
 trap - INT TERM
 
 docker run --rm \
-	-t $(tty &>/dev/null && echo "-i") \
```

Comment:
This is very similar to what was there before, but now it's POSIX compliant (technically doesn't require Bash)